velox/exec/fuzzer/WindowFuzzer.cpp

```diff
@@ -60,7 +60,9 @@ void WindowFuzzer::addWindowFunctionSignatures(
   }
 }
 
-std::tuple<std::string, bool> WindowFuzzer::generateFrameClause() {
+std::tuple<std::string, bool> WindowFuzzer::generateFrameClause(
+    std::vector<std::string>& argNames,
+    std::vector<TypePtr>& argTypes) {
```

Comment:
nit: moving the bool as an out parameter like argNames and argTypes would make it more consistent.